metal3-dev/local-cbo.sh

```diff
@@ -8,6 +8,21 @@ source $SCRIPTDIR/logging.sh
 source $SCRIPTDIR/common.sh
 source $SCRIPTDIR/utils.sh
 
+# Stop already running CBO.
+pushd $SCRIPTDIR
+$SCRIPTDIR/stop-cbo.sh
+
+CBO_LOCAL_IMAGE_PATH="/etc/cluster-baremetal-operator/images"
+
+# Copy cbo-images.json into the CBO_LOCAL_IMAGE_PATH to run CBO locally. 
+# CBO needs image.json file in this path.
+if [[ ! -f "${CBO_LOCAL_IMAGE_PATH}/images.json" ]]; then
+    sudo mkdir -p $CBO_LOCAL_IMAGE_PATH
+    sudo cp ${OCP_DIR}/cbo-images.json ${CBO_LOCAL_IMAGE_PATH}/images.json
+fi
```

Comment:
What about instead passing directly to the make the location of the files? Ie

`make run -e ${OCP_DIR}/cbo-images.json`

This should save some code here

Reply:
Agree, that would save some code.